src/app/views/sap/create_mission.php

```diff
@@ -15,10 +15,10 @@
                 <?php
                 if (isset($errors) && count($errors) !== 0) {
                     foreach ($errors as $error) {
-                        echo "<p style=\"color:red\">$error</p>";
+                        echo "<p class=\"text-error\">$error</p>";
```

Comment:
That's actually the fun part. :P For some reason I hate string concatenation when variable insertion is possible. We'll discuss this later though. Not import right now, plus, it's not being introduced in this PR.